src/sagemaker/djl_inference/model.py

```diff
@@ -151,26 +153,52 @@ def _get_model_config_properties_from_s3(model_s3_uri: str):
     return model_config
 
 
+def _get_model_config_properties_from_hf(model_id: str):
+    """Placeholder docstring"""
+
+    git_lfs_model_repo = "https://huggingface.co/" + model_id
+    my_env = os.environ.copy()
+    my_env["GIT_LFS_SKIP_SMUDGE"] = "1"
+    my_env["GIT_TERMINAL_PROMPT"] = "0"
+    model_config = None
+    with TemporaryDirectory() as tmp_dir:
+        subprocess.check_call(["git", "clone", git_lfs_model_repo, tmp_dir], env=my_env)
```

Comment:
No, the GIT_LFS_SKIP_SMUDGE=1 will cause git to download pointers to the large files. This is still a bit overkill since it will download all the json/txt files. 

I don't see any built in utils to just download a single file from git, but another option could be
`curl https://huggingface.co/<repo/model>/raw/main/config.json > config.json`

I chose the git option (though likely not the optimal implementation) since it this sdk already assumes that the user has git installed wherever they are running the code. curl or wget is more efficient to just get the single file, but i'm not sure whether we should assume user has that installed wherever they are running this from. Thoughts?